extensions/omnivore/src/omnivore-capture.tsx

```diff
@@ -0,0 +1,29 @@
+import { LaunchProps, Toast, showToast } from '@raycast/api'
+import { saveUrl } from './utils'
+
+export default async function QuickCapture(props: LaunchProps<{ arguments: Arguments.OmnivoreCapture }>) {
+  const { url, labels } = props.arguments
+
+  showToast({
+    style: Toast.Style.Animated,
+    title: 'Saving Article',
+  })
+
+  try {
+    const isUrlSaved = await saveUrl(url, labels)
+    if (isUrlSaved) {
+      showToast({
+        style: Toast.Style.Success,
+        title: 'Saved!',
+        message: 'Your URL was saved',
+      })
+    }
```

Comment:
Maybe add a failure toast here if `isUrlSaved` is `false`

Reply:
Agree, actually because how the saveUrl function is constructed (it returns true or false always) we don't need try/catch block here - if/else block should be enough and it should simplify the code.